File: Java_Basic_Jan_2023_SolutionsOfProblems/P02AdditionalProblems_ConditionalStatements/P06_FlowerShop.java
package P02AdditionalProblems_ConditionalStatements;

import java.util.Scanner;

public class P06_FlowerShop {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int magnolias = Integer.parseInt(scanner.nextLine());
        int hyacinths = Integer.parseInt(scanner.nextLine());
        int roses = Integer.parseInt(scanner.nextLine());
        int cactus = Integer.parseInt(scanner.nextLine());
        double giftPrice = Double.parseDouble(scanner.nextLine());

        double amount = (magnolias * 3.25 + hyacinths * 4 + roses * 3.5 + cactus * 8) * 0.95;

        if (giftPrice <= amount)
            System.out.printf("She is left with %.0f leva.", Math.floor(amount - giftPrice));
        else
            System.out.printf("She will have to borrow %.0f leva.", Math.ceil(giftPrice - amount));
    }

}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.MoreEx_JS_Conditional_Stitiments-More Exercises/P06_FlowerShop.js
function flowerShop(input) {
    
    let magnoliasNumber = Number(input[0]);
    let springFlowersNumber = Number(input[1]);
    let rosesNumber = Number(input[2]);
    let cactiNumber = Number(input[3]);
    let giftCost = Number(input[4]);

    let profit = (magnoliasNumber * 3.25 + springFlowersNumber * 4 + rosesNumber * 3.5 + cactiNumber * 8) * 0.95;

    if (profit < giftCost) {
        console.log(`She will have to borrow ${Math.ceil(giftCost - profit)} leva.`);
    } else {
        console.log(`She is left with ${Math.floor(profit - giftCost)} leva.`);
    }
}